Dockerfile

```diff
@@ -2,7 +2,6 @@ FROM hydroshare/hs_docker_base:release-1.9
 MAINTAINER Phuong Doan pdoan@cuahsi.org
 
 # inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
-RUN pip uninstall -y django-inplaceedit
```

Comment:
@hyi - I was able to remove this.  My previous test must've been using an older hs_docker_base image.